### ISSUE TITLE
Help

### ISSUE THREAD

**Issue by gfxanuj123** [brentN5/bt-target#25]
How can i add a event which is like  this TriggerEvent("server-inventory-open", "1", "Shop")
When adding this on event field its not Working  :- 

options = {
         {
                   event = "server-inventory-open","1","Shop",
                   icon = "fas fa-coffe",
                   label = "shop",
},
},

**Comment by LiamDormon** [comment]
Correct, that's now how it works

**Comment by gfxanuj123** [comment]
Ok then thanks for your reply

**Comment by gfxanuj123** [comment]
But is there any way to make them work?

**Comment by LiamDormon** [comment]
No, it doesn't trigger an event with parameters, you need to set it up properly
